src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/CompletionSource.cs

```diff
@@ -248,6 +257,9 @@ public async Task<object> GetDescriptionAsync(IAsyncCompletionSession session, V
             return IntelliSense.Helpers.BuildClassifiedTextElement(description.TaggedParts);
         }
 
+        public OptionSet GetOptions(Workspace workspace)
+            => _isDebuggerTextView ? workspace.Options.WithDebuggerCompletionOptions() : workspace.Options;
```

Comment:
This is fine especially since it's mirroring what we're already doing in the Controller, but it's possible for a workspace to override options so workspace.Options does the right thing without having to couple these components together.

Want to file a bug on this (or just fix it in master?) This will probably also remove @AmadeusW's questions about "why is this magic" because it doesn't have to be magic here.